mappings/net/minecraft/entity/EntityGroup.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/class_2280 net/minecraft/entity/EntityGroup
+	FIELD field_33254 DEFAULT Lnet/minecraft/class_2280;
```

Comment:
is this an enum
if it is dont rename it
`genSources` will scream

Reply:
Upstream does it too
https://github.com/FabricMC/yarn/blob/20w51a/mappings/net/minecraft/entity/EntityGroup.mapping